packages/typescript-estree/tests/ast-alignment/parse.ts

```diff
@@ -96,7 +98,7 @@ export function parse(
         );
     }
   } catch (error) {
```

Comment:
We could use the `unknown` annotation here
But it'd make some of the other stuff a bit harder.
Maybe it's just better to keep the disable on L117?

```suggestion
  } catch (error: unknown) {
```

Reply:
yeah, this is a pain :( multiple places get type checked as unknown....